.circleci/config.yml

```diff
@@ -7170,14 +7170,14 @@ workflows:
             - pytorch_linux_bionic_py3_8_gcc9_coverage_build
           build_environment: "pytorch-linux-bionic-py3.8-gcc9-coverage-test1"
           docker_image: "308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-bionic-py3.8-gcc9"
-          resource_class: large
+          resource_class: xlarge
```

Comment:
@seemethere @malfet @ezyang @janeyx99 

BTW, how to verify if AVX512 is supported on either `large` or `xlarge`?

According to circleci's pricing plan, it's a 5x cost increase ([link](https://circleci.com/pricing/?utm_source=gb&utm_medium=SEM&utm_campaign=SEM-gb-200-Eng-uscan&utm_content=SEM-gb-200-Eng-uscan-CirclePricing&utm_term=a2&gclid=CjwKCAjw8cCGBhB6EiwAgORey9k-uogWI-mo6bx8kUQfh8WaEGvqyX2AhDZWniTXDoSMEfwzmtEbEhoC2RUQAvD_BwE)). Might be helpful if it's also validated on GHA's runner `linux.2xlarge`, and we've got cost-efficient hardware there. Related migration process https://github.com/pytorch/pytorch/issues/57686

Reply:
> how to verify if AVX512 is supported on either `large` or `xlarge`?

@zhouzhuojie, I had used `lscpu` in #59100 to verify that `xlarge` instances have AVX512 support.
On `large` instances, the default `ATEN_CPU_CAPABILITY` for this PR was printed in the logs as AVX2, so that means `large` instances don't support AVX512.

> Might be helpful if it's also validated on GHA's runner linux.2xlarge

`linux.2xlarge` on GHA definitely has AVX512 support, as it's an AWS [`c5.2xlarge`](https://aws.amazon.com/ec2/instance-types/c5/) instance.